infra/terraform/global/main.tf

```diff
@@ -48,3 +48,21 @@ module "log_pruning" {
       days: 30
 EOF
 }
+
+// Backup etcd volumes attached to kubernetes masters -->
+data "archive_file" "kubernetes_etcd_ebs_snapshot_code" {
+  source_file = "assets/create_etcd_ebs_snapshot/create_etcd_ebs_snapshot"
```

Comment:
Are we checking in the output/binary in Git? (which is a bit of an antipattern as in theory it's derived from the source code)

I'm asking because I'm wondering how would work if someone without this file tries to do a `terraform apply`, or if someone doesn't have a working Go environment or it's not familiar with Go, etc...